README.md

```diff
@@ -190,4 +187,4 @@ We are glad to review and accept contributions.
 
 Today, the scope is limited to enabling/optimizing VMs on public cloud providers.
```

Comment:
Can we modify this a bit, as I think we'll want to use this repo to provide some help with Gaudi configurations.